src/utils/hooks.ts

```diff
@@ -7,3 +8,15 @@ export const usePrevious = (value: any) => {
   });
   return ref.current;
 };
+
+export const useForsideInnhold = (stønadstype: string): any => {
```

Comment:
Det har ikke så mye å si, men kanskje man kunne bruk `enum` som type i stedet for `string` her? :) 
Så vil vi få samlet alle stønadstypene som går an å kalle på :)

Reply:
Lurt!